src/js/dynload.ts

```diff
@@ -32,7 +32,7 @@ function createDynlibFS(
   readFileFunc?: ReadFileType,
 ): LoadDynlibFS {
   const dirname = lib.substring(0, lib.lastIndexOf("/"));
-  const defaultSearchDirs = [API.dsodir, API.sitepackages, dirname];
+  const defaultSearchDirs = API.defaultLdLibraryPath + [dirname];
```

Comment:
You can't add lists like this in Typescript can you? You are probably thinking of Python.

Reply:
Good point, Thanks! It seems like this was the reason why I was getting mysterious errors 😂